Makefile

```
@@ -27,4 +27,4 @@ testsetup:
 	echo "running rosdep tests"
 
 test: testsetup
-	cd test && pytest
+	pytest
```

I think it would still be safer to limit the pytest collection to only the `test` subdirectory:

```suggestion
	pytest test
```